### PR TITLE
[TEST PR - DNM] Nonetns

### PR DESCRIPTION
**Pull request by caoruidong**
This is a test PR. Since last pr got some errors I can't understand. I test this to find out the reason.

### CODE REVIEW

File: virtcontainers/network.go [hunk 1]
```diff
@@ -832,6 +838,12 @@ func doNetNS(netNSPath string, cb func(ns.NetNS) error) error {
 	}
 	defer currentNS.Close()
 
+	// if netNSPath is empty, the callback function will be run in the current network namespace.
+	// So skip useless GetNS and Set, just call cb().
+	if netNSPath == "" {
```
[the remark this answers is not in the document]
Comment (caoruidong) [reply virtcontainers/network.go]: OK, exactly. @bergwolf Thanks for your review. I'll address them in original PR. This pr is just a test and will be closed after I find why CI fails.

**Comment by jodh-intel** [comment]
Hi @caoruidong. I understand from https://github.com/kata-containers/runtime/pull/835/files#r226216859 that this is a test PR, based on #733.

Raising test PRis is fine, but it would help users if you could explain this to avoid confusion :)

I'll update the title of this PR to make it clearer...

**Comment by caoruidong** [comment]
@jodh-intel Thank you. I will try to not open test pr anymore :)

**Comment by jodh-intel** [comment]
Hi @caoruidong - please *do* raise test PRs if you need to :) All I mean is that we have so many PRs "flying around", it helps a lot if you link back to the original PR and make it clear your PR is a test PR.

**Comment by caoruidong** [comment]
original PR: https://github.com/kata-containers/runtime/pull/733

**Comment by caoruidong** [comment]
@jodh-intel Yeah. I mean I don't want to meet CI problem once more…

**Pull request closed without merging**